app/assets/javascripts/assignments.js

```diff
@@ -250,10 +262,39 @@
     $(".file-picker").each(function(index) { activate_file_picker($(this)); });
   }
 
+  function toggles_init() {
+    $.each($('.toggle-wrapper'), function(index, toggleWrapper) {
+      $(toggleWrapper).click(function(e) {
```

Comment:
Even more idiomatically, you can just say `$('.toggle-wrapper').click(function(e) { ... })`, which will apply the function as a click event handler to each of the matched elements.  Within the callback function, `this` will be bound to the element that was clicked.

Reply:
Good to know, thanks